### PR TITLE
[CF-1142] Changes to run tiamat for feeds configured in conf file by …

### PR DESCRIPTION
**Pull request by ChandraAddala**
…default

### CODE REVIEW

File: tiamat/src/main/scala/com/rackspace/feeds/archives/Options.scala [hunk 1]
```diff
@@ -224,15 +224,14 @@ class Options {
...
-      }
       case r: RunConfig if r.isTestMode => {
 
         r.copy( feeds = r.getTestFeeds() )
       }
-      case r: RunConfig => r
+      case r: RunConfig =>  {
+
+        r.copy(feeds = feedsConf.toList )
+      }
```
Comment (shintasmith) [inline tiamat/src/main/scala/com/rackspace/feeds/archives/Options.scala]: Will this undo the change we put in place to not make Tiamat runs all tenants by default?

Comment (shintasmith) [inline tiamat/src/main/scala/com/rackspace/feeds/archives/Options.scala]: @usnavi, pls review this. I don't quite understand the changes. I am hoping you do. :-)

Comment (usnavi) [inline tiamat/src/main/scala/com/rackspace/feeds/archives/Options.scala]: @shintasmith - yeah, something didn't look right on the -f either.  I believe the unit test for -f is incorrect and @ChandraAddala  was going to take a look.

Comment (ChandraAddala) [inline tiamat/src/main/scala/com/rackspace/feeds/archives/Options.scala]: @shintasmith This change will not break the --all-tenants option which is handled in processTenants method but it did break the -f option. I made another commit for this. I fixed the test as well which should have caught this.

**Comment by usnavi** [comment]
lgtm - thanks @ChandraAddala

**Comment by shintasmith** [comment]
looks good
